octobot_trading/personal_data/orders/order.py

```diff
@@ -44,6 +44,7 @@ def __init__(self, trader, side=None):
         self.is_synchronized_with_exchange = False
         self.is_from_this_octobot = True
         self.simulated = trader.simulate
+        self.allow_self_managed = True
```

Comment:
I don't understand this attribute, more precisely why it has to be an order attribute?

Reply:
move into trading config "allow_artificial_orders" => stored into trader